dashbuilder/dashbuilder-backend/dashbuilder-external-backend/src/main/java/org/dashbuilder/external/impl/ComponentServiceImpl.java

```diff
@@ -55,11 +56,13 @@
     
     @Override
     public List<ExternalComponent> listAllComponents() {
-        List<ExternalComponent> allComponents = loader.loadExternal();
+        List<ExternalComponent> allComponents = new ArrayList<>();
+        List<ExternalComponent> external = loader.loadExternal();
         List<ExternalComponent> provided = loader.loadProvided();
 
-        allComponents.forEach(c -> c.setProvided(false));
+        external.forEach(c -> c.setProvided(false));
         provided.forEach(c -> c.setProvided(true));
+        
         allComponents.addAll(provided);
```

Comment:
```suggestion
        allComponents.addAll(provided);
        allComponents.addAll(external);
```

Reply:
Thanks again! This was the same fix for the bug you reviewed in other PR - the bug on a bug fix would be here  again - fixed now